README.md

```
@@ -68,3 +69,28 @@
    git clone https://github.com/vladimirbalaur18/xLineDesignWeb.git
    cd xLineDesignWeb
    ```
+
+### Deploying on CloudFlare Workers and Pages
+
+For CloudFlare deployment, wrangler.toml is the configuration file.
+
+1. You must login using wrangler before running the `wrangler` CLI (`wrangler login`)
+2. Run the commands via `wrangler`, (` wrangler build`, `wrangler dev`) since the `opennextjs-cloudflare` from official documentation doesn't work
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)